.gitignore

```diff
@@ -0,0 +1,3 @@
+__pycache__/
+.idea/
+.vscode/
```

Comment:
While it's harmless to populate a project-local .gitignore file with content like this, you'll find yourself duplicating the patterns in every project you use an IDE with. Rather than repeating the same patterns in each project you touch, I suggest creating a global gitignore file that excludes patterns like these. [This blog post](https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/) does a good job describing the steps. This way you can ignore `.idea` or `.vscode` in *every* project you touch.